src/mui/button/RefreshButton.js

```diff
@@ -1,20 +1,28 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import compose from 'recompose/compose';
 import FlatButton from 'material-ui/FlatButton';
 import NavigationRefresh from 'material-ui/svg-icons/navigation/refresh';
 import translate from '../../i18n/translate';
+import { refreshList as refreshListAction } from '../../actions/listActions';
 
-const RefreshButton = ({ label = 'aor.action.refresh', translate, refresh }) => <FlatButton
+const RefreshButton = ({ label = 'aor.action.refresh', translate, refresh, resource }) => <FlatButton
     primary
     label={label && translate(label)}
-    onClick={refresh}
+    onClick={() => refresh(resource)}
```

Comment:
Don't declare functions inside JSX as it will change at each render. You can use [recompose/withHandler](https://github.com/acdlite/recompose/blob/master/docs/API.md#withhandlers) to inject a preconfigured function in the props and use it directly in the onClick handler.